### PR TITLE
tuhi-exam-vuw:0.1.0

### PR DESCRIPTION
**Pull request by baptiste**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This template follows the formatting of VUW exam scripts. It also defines a `#mark()` function to set the marks of each question, and a `#fin` macro to end the script with the required `* * *` asterisks.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/tuhi-exam-vuw/0.1.0/template/main.pdf
Comment (elegaanz) [inline packages/preview/tuhi-exam-vuw/0.1.0/template/main.pdf:1]: I think this file should be deleted.

File: packages/preview/tuhi-exam-vuw/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tuhi-exam-vuw"
```
Comment (elegaanz) [inline packages/preview/tuhi-exam-vuw/0.1.0/typst.toml:2]: The name of the package should not be the most obvious/canonical one for what the package does. Can you please rename it to something else?

File: packages/preview/tuhi-exam-vuw/0.1.0/vuw-logo.svg
Comment (elegaanz) [inline packages/preview/tuhi-exam-vuw/0.1.0/vuw-logo.svg:1]: Is this logo under a license that allows it to be copied in this repository?

File: packages/preview/tuhi-exam-vuw/0.1.0/typst.toml (hunk 1 above)
Comment (baptiste) [reply packages/preview/tuhi-exam-vuw/0.1.0/typst.toml:2]: I was hoping to have at least some consistency with [tuhi-course-poster-vuw](https://typst.app/universe/package/tuhi-course-poster-vuw). Note that `tuhi` is a Maori word that is never used in English, so I really can't imagine anyone would come up with this same package name, or mistake it for a canonical name.

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**